Druid/DRCPSEdge.class.st

```diff
@@ -94,7 +94,9 @@ DRCPSEdge >> integerOperation: anOperantion [
 
 { #category : #testing }
 DRCPSEdge >> isDead [
-	^ constraints anySatisfy: [ :c | c isEmpty ]
+
+	^ constraints notEmpty and: [
+		  constraints allSatisfy: [ :c | c isEmpty ] ]
```

Comment:
@PalumboN please take a look at this line

Reply:
Mmmm, I have doubts about it. If I remember well:

- One edge (the connection between 2 blocks) has the info about all possible values (or constraints) for each register at some point in the program (after the `source` block) inside a path.
- If one register has no possible values (is empty) then this edge is dead because exist one register without possible values for it.
- Could be other registers (for example constants copies) that will never be empty.

But, strangely, all tests are green... maybe some case for this (mixing dead and not dead) is missing...